### ISSUE TITLE
Data copy between different sources

### ISSUE THREAD

**Issue by win32nipuh** [microsoft/azuredatastudio#8963]
I have 2 connections: SQL Server and PostgreSQL and I need to transfer data from PostgreSQL DB1 to SQL Server DB2 like I can do it in SSMS using linked server.

How to do it in Azure Data Studio?

**Comment by Charles-Gagnon** [comment]
You can execute the queries against linked servers in ADS - is there a particular thing you're blocked on?

You could also look into dumping the data into a CSV and then using the [Import Wizard](https://docs.microsoft.com/en-us/sql/azure-data-studio/sql-server-import-extension?view=sql-server-ver15) to import that file into the SQL table. 

Although if you're transferring data usually it's better to use a tool specifically designed for that - such as SSIS - since the performance is going to be a lot better in those applications.

**Comment by win32nipuh** [comment]
> You can execute the queries against linked servers in ADS - is there a particular thing you're blocked on?
> 
> You could also look into dumping the data into a CSV and then using the [Import Wizard](https://docs.microsoft.com/en-us/sql/azure-data-studio/sql-server-import-extension?view=sql-server-ver15) to import that file into the SQL table.
> 
> Although if you're transferring data usually it's better to use a tool specifically designed for that - such as SSIS - since the performance is going to be a lot better in those applications.

Yes, you are right, This way SSMS works, but ADS is the new tool and it has connections to both sources, Thats why it may be useful feature to work with both sources do not using linked servers. 

For example, SQL Server is installed on Ubuntu, PostgreSQL is installed on Mac OS and ADS works in linux too. I work with these 2 sources.
